WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/navigation/OrderCreateEditNavigationTarget.kt

```diff
@@ -32,6 +34,11 @@ sealed class OrderCreateEditNavigationTarget : Event() {
         val couponCode: String? = null
     ) : OrderCreateEditNavigationTarget()
 
+    data class AddCoupon(
```

Comment:
```suggestion
    data class SelectCoupon(
```
What do you think about renaming it?

Reply:
removed [cdac4f3](https://github.com/woocommerce/woocommerce-android/pull/9510/commits/cdac4f30ca6fc79599959fb35be25bfbeee29adb)

Reply:
I thought about it, but then, this is regarding the button in the Order Creation screen. The button says `+Add Coupon` so I thought it would be confusing. SelectCoupon to me would be (as it was before we decided to use ExitWithResults) the data class regarding the navigation from the coupon selector list back to Order creation. Does it make sense? Wdyt?

Reply:
Makes sense! Thx for sharing your thoughts :)